### PR TITLE
Upgrade to latest stable lodash

### PR DESCRIPTION
**Pull request by gigabo**
The new version doesn't support modularization in the same way as our old
version.  So, here, I've just abandoned it.

This adds ~20KB to minified compressed bundle.  It's simpler.  Worth it?

### CODE REVIEW

**Comment by roblg** [comment]
I was under the impression that lodash v4 was supposed to be _better_ at modularization that v3? 

I can't link into headers in release notes, but... https://github.com/lodash/lodash/releases/tag/4.0.0

**Comment by gigabo** [comment]
Oh... huh. :)

**Comment by gigabo** [comment]
Okay, pushed a version that's modularized.

Still ~4KB bigger, but probably worth it to be off of creaky ancient technology.

**Comment by doug-wade** [comment]
lgtm

http://i.imgur.com/vqD8jZh.gif

**Comment by roblg** [comment]
👍

File: packages/react-server/package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "cookie-parser": "1.3.3",
     "eventemitter3": "0.1.6",
     "express-state": "1.2.0",
-    "lodash": "3.10.1",
+    "lodash": "^4.12.0",
```
Comment (roblg) [inline packages/react-server/package.json]: Just out of curiosity, did we intentionally unpin this? ("Yes" is a reasonable answer, just curious.)

Comment (gigabo) [inline packages/react-server/package.json]: Nope, just did an `npm install --save lodash@latest`.

I'm not as paranoid about pinning as I once was.  I'd actually like to automatically pick up patch-level changes.

Unless they break everything.  That would suck.

**Pull request merged**